### ISSUE TITLE
Need to apply ed4718ec

### ISSUE THREAD

**Issue by kaz7** [sx-aurora-dev/llvm-project#33]
@simoll -san, can you check ed4718ec?  I want to merge upstream, but this patch in upstream adds one line to operand-bundles-bc-analyzer.ll like below.
```
diff --git a/llvm/test/Bitcode/operand-bundles-bc-analyzer.ll b/llvm/test/Bitcode/operand-bundles-bc-analyzer.ll
index f22fcbeb271e..cb7a3ee7eb71 100644
--- a/llvm/test/Bitcode/operand-bundles-bc-analyzer.ll
+++ b/llvm/test/Bitcode/operand-bundles-bc-analyzer.ll
@@ -9,6 +9,7 @@
 ; CHECK-NEXT:    <OPERAND_BUNDLE_TAG
 ; CHECK-NEXT:    <OPERAND_BUNDLE_TAG
 ; CHECK-NEXT:    <OPERAND_BUNDLE_TAG
+; CHECK-NEXT:    <OPERAND_BUNDLE_TAG
 ; CHECK-NEXT:  </OPERAND_BUNDLE_TAGS_BLOCK

 ; CHECK:   <FUNCTION_BLOCK
```
On the other hand, your modifications on e58723f adds two lines like below, and this one is already applied to our develop branch.
```
diff --git a/llvm/test/Bitcode/operand-bundles-bc-analyzer.ll b/llvm/test/Bitcode/operand-bundles-bc-analyzer.ll
index f22fcbeb271e..1504f5b525db 100644
--- a/llvm/test/Bitcode/operand-bundles-bc-analyzer.ll
+++ b/llvm/test/Bitcode/operand-bundles-bc-analyzer.ll
@@ -9,6 +9,8 @@
 ; CHECK-NEXT:    <OPERAND_BUNDLE_TAG
 ; CHECK-NEXT:    <OPERAND_BUNDLE_TAG
 ; CHECK-NEXT:    <OPERAND_BUNDLE_TAG
+; CHECK-NEXT:    <OPERAND_BUNDLE_TAG
+; CHECK-NEXT:    <OPERAND_BUNDLE_TAG
 ; CHECK-NEXT:  </OPERAND_BUNDLE_TAGS_BLOCK

 ; CHECK:   <FUNCTION_BLOCK
```
The problem is that if I try to merge upstream, git ask me to add either only one line or none.  However, I guess you may need 3 lines after upstream's modifications since upstream want to add 1 and you want to add 2.  Can you check it by yourself?  There are several similar conflicts in upstream.  It's annoying.

**Comment by kaz7** [comment]
Please use 2a33b54 for your check.  I've merged 81b1d3d which is one previous patch of ed4718e.  You can check it by following commands:
```
$ git checkout 2a33b54 
$ git merge ed4718e
```

**Comment by simoll** [comment]
Yes, there should be three new entries:
LLVM for VE adds operand bundles for fp except and fp rounding mode specification (https://reviews.llvm.org/D93455).
Another bundle type was added in upstream since February.

**Comment by kaz7** [comment]
Thank you!  I'll add three then.  There are several similar modifications on upstream.  I'll do the similar modifications on others also.

**Comment by jam7** [comment]
The resolution of this issue is merged by #35.  So, I'm closing this.
